internal/gardenclient/client_test.go

```diff
@@ -64,6 +119,8 @@ var _ = Describe("Client", func() {
 		},
 			Entry("and the secretName has suffix .oidc", "seed-1"),
 			Entry("and the secretName has suffix .login", "seed-2"),
+			Entry("and seed is a managed seed, return shoot client config", "managedSeed-1"),
```

Comment:
This should not be added to this table test. If we have a managed seed the secret doesn't matter. may be for this case anyway the table test make not much sense. I would prefer to use a helper with ExpectOffset and create or override the resources for each test as needed.